### PR TITLE
{bio}(foss/2015b) Cufflinks 2.2.1, SAMtools 0.1.20 (REVIEW)

### PR DESCRIPTION
**Pull request by nathanhaigh**
New Cufflinks PR following stuff-up in #2279

### CODE REVIEW

**Comment by boegel** [comment]
Jenkins: ok to test

File: easybuild/easyconfigs/c/Cufflinks/Cufflinks-2.2.1-foss-2015b.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+dependencies = [
+    ('Boost', '1.60.0', '-Python-2.7.10'),
+    ('SAMtools', '0.1.20'),
+    ('Eigen', '3.2.7'),
+    ('zlib', '1.2.8'),
+]
+
+preconfigopts = 'CPPFLAGS=-I$EBROOTEIGEN/include'
+configopts = '--with-boost=$EBROOTBOOST --with-bam=$EBROOTSAMTOOLS'
```
Comment (boegel) [inline easybuild/easyconfigs/c/Cufflinks/Cufflinks-2.2.1-foss-2015b.eb]: existing easyconfig for Cufflinks v2.2.1 uses ` --with-bam-libdir=${EBROOTSAMTOOLS}/lib`?

File: easybuild/easyconfigs/s/SAMtools/SAMtools-0.1.20-foss-2015b.eb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+name = 'SAMtools'
+version = '0.1.20'
+
+homepage = 'http://www.htslib.org/'
+description = """SAM Tools provide various utilities for manipulating alignments in the SAM format,
+ including sorting, merging, indexing and generating alignments in a per-position format."""
+
+toolchain = {'name': 'foss', 'version': '2015b'}
+toolchainopts = {'optarch': True, 'pic': True}
```
Comment (boegel) [inline easybuild/easyconfigs/s/SAMtools/SAMtools-0.1.20-foss-2015b.eb]: `optarch` is enabled by default, no need to hardcode it here

existing `SAMtools` easyconfig don't have `pic` enabled, should we do it for all of them? (not in this PR please)

File: easybuild/easyconfigs/s/SAMtools/SAMtools-0.1.20-foss-2015b.eb [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+source_urls = ['https://github.com/samtools/samtools/archive']
+sources = ['%(version)s.tar.gz']
```
Comment (boegel) [inline easybuild/easyconfigs/s/SAMtools/SAMtools-0.1.20-foss-2015b.eb]: this is slightly different from existing `SAMtools` 0.1.20 easyconfigs, any particular reason?

also, the existing easyconfigs include a patch related to `ncurses`, why not here?

**Comment by boegel** [comment]
missing deps:

```
Irresolvable dependencies encountered: Boost/1.60.0-foss-2015b-Python-2.7.10, Eigen/3.2.7-foss-2015b
```

**Comment by boegel** [comment]
@nathanhaigh ping on missing deps

**Comment by migueldiascosta** [comment]
@nathanhaigh, this PR is being closed for the following reason(s): no activity for > 6 months, obsoleted by more recent PRs.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**